### ISSUE TITLE
"sh -c echo workflow not found && exit 1" running job

### ISSUE THREAD

**Issue by marcosbianchi** [aweris/gale#114]
Workflows listed ok using: `gale list`

Executed `gale run .github/workflows/workflow.yml  job-name`

```
76: exec /entrypoint
76: > in service v975nob2u91fo.8951mv3r8eggq.dagger.local
Error: input:1: container.from.withUnixSocket.withFile.withEnvVariable.withMountedCache.withServiceBinding.withEnvVariable.withEnvVariable.withServiceBinding.withEnvVariable.withEnvVariable.withExec.sync process "sh -c echo workflow not found && exit 1" did not complete successfully: exit code: 1
```

**Comment by aweris** [comment]
`gale` isn't able to find the workflow you want to execute. Is it possible to share the `list` output? 

It is possible that the workflow you are trying to call has a name, but you are attempting to call it using the workflow path instead. The workflow path is a fallback value that is used when the workflow name is missing or if you have made a typo in the workflow path when calling the command

**Comment by marcosbianchi** [comment]
Sure!

![Screenshot 2023-09-02 at 13 54 40](https://github.com/aweris/gale/assets/12137035/32d4daa9-cfc2-4950-afc4-c5b5c109d1d5)

![Screenshot 2023-09-02 at 13 57 38](https://github.com/aweris/gale/assets/12137035/6575aa2c-5fba-4ecb-84c4-ec179fc15576)

**Comment by aweris** [comment]
it seems your workflow name is `Lint` but you're trying to call workflow with its path. Can you try to run `gale run Lint run-linters` 

<img width="541" alt="image" src="https://github.com/aweris/gale/assets/9319656/6cdb470d-a659-4317-bcd4-0221618c9fac">

**Comment by marcosbianchi** [comment]
Yeah, your're right, that worked!  I always though in the filename instead of the name. 
Thank your very much for your time

<img width="1000" alt="image" src="https://github.com/aweris/gale/assets/12137035/6d0492a0-46b7-4916-b86e-0b9ef6362378">

**Comment by aweris** [comment]
That's perfect. I'm closing this issue then. 

 I'm seeing an error in your pipeline, if it's open source can you share this project with me? I would like to know the cause of the error.
